amqp/pom.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>io.cloudevents</groupId>
+		<artifactId>cloudevents-parent</artifactId>
+		<version>2.0.0-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>cloudevents-amqp</artifactId>
```

Comment:
should we name it following the same pattern of other "integration modules" aka `cloudevents-[protocol]-[library]`? Or better, do we expect someday someone comes in proposing a new amqp integration with another library (eg http integrations)? Or is it proton-j a standard de facto client, pretty much like official java kafka client is the standard de facto client for kafka?

Reply:
good question! i dont know the answer TBH. 

i would think of qpid-proton more as a library that can be integrated in any AMQP client/broker/etc. One can use a [`JMS`](https://www.oracle.com/technical-resources/articles/java/intro-java-message-service.html) client, or the rabbit-mq client and there might be other clients too. these clients may not rely on proton-j though.

so i think we can use `cloudevents-[protocol]-[library]`, WDYT?

Reply:
yeah let's go with `cloudevents-amqp-proton` ok?

Reply:
sure, should i then change the name of the class to something with `proton` in the name? e.g `ProtonBasedAmqpBinaryMessageWriter`?